sphinx/source/index.rst

```diff
@@ -1,6 +1,8 @@
 Welcome to Bokeh
 ================
 
+**A framework for interactive browser-based visualizations**
+
```

Comment:
I'm incredibly loathe to use the word "framework" anywhere. It's laden with baggage and meanings we surely do not intend. "platform"  is probably a better choice but then the text below should also be updated accordingly.  I'm also not sure about how this will look visually, can you attach a screen shot?